### PR TITLE
use handle DeletedFinalStateUnknown objects in function deleteNode

### PR DESCRIPTION
**Pull request by hanxiaoshuai**
**What this PR does / why we need it**:
fixtodo:ensure len(nodeName) > 0 in function deleteNode
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by hanxiaoshuai** [comment]
cc @mtaufen

File: plugin/pkg/auth/authorizer/node/graph_populator.go [hunk 1]
```diff
@@ -116,12 +116,12 @@ func (g *graphPopulator) updateNode(oldObj, obj interface{}) {
 func (g *graphPopulator) deleteNode(obj interface{}) {
 	node := obj.(*api.Node)
```
Comment (liggitt) [inline plugin/pkg/auth/authorizer/node/graph_populator.go]: it's not necessary to check the node name... if we get a delete event for a zero-length node name, it is harmless in this case, and we have much bigger problems.

however, while you're in here, can you correct this to handle DeletedFinalStateUnknown objects:

```go
func (g *graphPopulator) deleteNode(obj interface{}) {
	if tombstone, ok := obj.(cache.DeletedFinalStateUnknown); ok {
		obj = tombstone.Obj
	}
	node, ok := obj.(*api.Node)
	if !ok {
		glog.Infof("unexpected type %T", obj)
		return
	}
	...
}
```

Comment (hanxiaoshuai) [reply plugin/pkg/auth/authorizer/node/graph_populator.go]: Done. Thanks for your quick review.

**Pull request merged**